src/senselab/text/tasks/emotional_analysis/api.py

```diff
@@ -0,0 +1,33 @@
+"""API module for emtoional analysis."""
+
+from typing import Dict, List, Optional, Union
+
+from senselab.text.tasks.emotional_analysis.emotional_analysis import EmotionalAnalysis
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import HFModel
+
+
+def analyze_emotion(
+    pieces_of_text: List[str],
+    device: Optional[DeviceType] = None,
+    model: Optional[HFModel] = None,
+    max_length: int = 512,
+    overlap: int = 128,
```

Comment:
`max_length` and `overlap`  look like model-specific params and shouldn't be in the general emotion recognition API. you can obtain them as part `** kwargs`